SnapshotTests/SnapshotTestCase.swift

```diff
@@ -93,6 +93,19 @@ extension SnapshotTestCase {
     }
 }
 
+extension SnapshotTestCase {
+    func retrieveRemoteConfiguration(_ fileName: String? = "MockConfiguration") -> RemoteConfiguration? {
```

Comment:
I think it is worth documenting this method, stating that this is used to test remote unified customisation.
Alternatively the method name could be changed to something like `retrieveRemoteConfigurationForUnifiedCustomization`.